enterprise/backend/src/metabase_enterprise/api/routes.clj

```diff
@@ -100,6 +102,7 @@
    "/database-routing"             (premium-handler metabase-enterprise.database-routing.api/routes :database-routing)
    "/document"                     (premium-handler metabase-enterprise.documents.api/routes :documents)
    "/email"                        (premium-handler metabase-enterprise.email.api/routes :cloud-custom-smtp)
+   "/embedding-hub"                (premium-handler metabase-enterprise.embedding-hub.api/routes :embedding)
```

Comment:
This makes sense, since we currently gate the embedding hub under `:embedding` and is likely to stay that way.